snap/snapcraft.yaml

```diff
@@ -239,7 +240,7 @@ parts:
       - locales-all
       # Landscape deps
       - postgresql-14-debversion=1.1.1-5
-      - postgresql-plpython3-14=14.11-0ubuntu0.22.04.1
+      - postgresql-plpython3-14
```

Comment:
We should move the pin here to `14.12-0ubuntu0.22.04.1`

Reply:
Sure. Pinned on https://github.com/canonical/charmed-postgresql-snap/pull/41/commits/c0472ff0d2edd596c9a3d550f76a5c28831566fc.